device-types/Supermicro/SuperStorage-6029P-E1CR16T.yaml

```diff
@@ -0,0 +1,26 @@
+---
+manufacturer: Supermicro
+model: SuperStorage 6029P-E1CR16T
+slug: supermicro-ssg-6029p-e1cr16t
+u_height: 2
+is_full_depth: true
+weight: 28.12
+weight_unit: kg
+airflow: front-to-rear
+comments: 'Specification: [https://www.supermicro.com/en/products/system/2U/6029/SSG-6029P-E1CR16T.cfm](https://www.supermicro.com/en/products/system/2U/6029/SSG-6029P-E1CR16T.cfm)'
+power-ports:
+  - name: PSU1
+    type: iec-60320-c14
+    maximum_draw: 1600
+  - name: PSU2
+    type: iec-60320-c14
+    maximum_draw: 1600
```

Comment:
These PSUs appear to be modular. If that is true, these should be defined as `module-bays` and the associated PSUs should be created in the module-types folder